mobileapps/views.py

```diff
@@ -1129,3 +1133,41 @@ def delete(self, request, theme_id):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class OrganizationThemeRemoveImageView(MobileRetrieveUpdateDestroyAPIView):
+    """
+    **Use Case**
+
+        Remove logo image or header background image from the theme.
+
+    **Example Requests**
+
+        DELETE /api/server/mobileapps//themes/{id}/remove/{attribute}
+
+        - attribute
+            - Logo Image: logo_image
+            - Header Background Image: header_bg_image
+    """
+
+    serializer_class = ThemeSerializer
+    lookup_url_kwarg = 'theme_id'
+
+    def __init__(self):
+        self.permission_classes += (IsStaffView,)
+
+    @transaction.atomic
+    def delete(self, request, theme_id, attribute):
+        theme = get_object_or_404(Theme, pk=theme_id)
+
+        if attribute == 'logo_image':
+            theme.remove_logo_image(
+                settings.ORGANIZATION_LOGO_IMAGE_SIZES_MAP,
```

Comment:
`remove_logo_image` and `remove_header_bg_image` methods encapsulate the data which is being passed via params.